javascript/javascript1/week1/housey.js

```diff
@@ -0,0 +1,12 @@
+let volumeInMeters =800;
+let houseCost = 2500000;
+let gardenSizeInM2 =100;
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+let housey;
+if(houseCost >= housePrice){
```

Comment:
We may not need conditionals here at all.

Reply:
Then , how to do it without conditions , can you send me an example of the code ?